.nvmrc

```diff
@@ -1 +1 @@
-lts/*
```

Comment:
Why is this pinned to this specific version and not 16 so we at least get some maintenance LTS updates?

Reply:
becuase i wanted the open ssl changes because it works with render anyways

Reply:
https://nodejs.org/en/blog/release/v16.16.0/